tests/test_dumpscript.py

```diff
@@ -78,22 +79,21 @@ def test_with_datetimefield(self):
             club=django,
         )
 
-        dumpscript_path = './django_extensions/scripts'
+        dumpscript_path = Path(__file__).parent.parent / 'django_extensions' / 'scripts'
+        dumpscript_path.mkdir(parents=True, exist_ok=True)
+        (dumpscript_path / '__init__.py').touch()
 
-        os.mkdir(dumpscript_path)
-        open(dumpscript_path + '/__init__.py', 'w').close()  # for python 2.7
+        # Delete dumpscript at the end of the test
+        self.addCleanup(shutil.rmtree, dumpscript_path)
 
         # This script will have a dateutil codes.
         # e.g. importer.locate_object(...,
         # 'date_joined': dateutil.parser.parse("2019-05-20T03:32:27.144586+09:00")
-        with open(dumpscript_path + '/test.py', 'wt') as test:
+        with (dumpscript_path / 'test.py').open('wt') as test:
             call_command('dumpscript', 'django_extensions', stdout=test)
 
         # Check dumpscript without exception
         call_command('runscript', 'test')
 
-        # Delete dumpscript
-        shutil.rmtree(dumpscript_path)
```

Comment:
I noticed that this didn't run and left some files on disk if one of the `call_command` above crashed.